.github/issue_template.md

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
This is oriented for bugs, what about features

Reply:
We can add more templates, let me work on it.